### DISCUSSION TITLE
Self-updater: Delta updates (already working)

### DISCUSSION THREAD

**Discussion by tutacat** [ppy/osu#30784] [Ideas] [answered]
Since a lot of the files will not change between updates, but osu! is about 150MB (depending on platform) it may be worth setting up delta updates, possibly with zsync

**Comment by felipemarins** [comment]
There was zsync files before Velopack was implemented and auto update started to work on Linux. Maybe it was removed because of that. I'm not sure if Velopack supports delta updates.

**Comment by smallketchup82** [comment, marked as answer]
Velopack already uses [delta updates](https://docs.velopack.io/packaging/deltas)

**Reply by felipemarins** [reply]
It's very likely that I'm misunderstanding it, but from what i got on the [GitHub section of this doc](https://docs.velopack.io/packaging/deltas#github-releases), it seems to imply that the release must have a delta update file (which the current release doesn't seem to have).

**Reply by smallketchup82** [reply]
The current release does have deltas for every platform. They are the files that end in -delta.nupkg. Which release did you check?

**Reply by felipemarins** [reply]
Oh, you're right. Sorry, I didn't see it when I was looking at the file names.
